src/BuildRetainer/Mono.Options/Options.cs

```diff
@@ -153,1214 +153,1311 @@ namespace NDesk.Options
 namespace Mono.Options
 #endif
 {
-	static class StringCoda {
```

Comment:
I do wonder if this BuildRetainer project is used any more?

Reply:
true. we just tell azdo to retain our signed builds.